### PR TITLE
Fix: The Action workflow install npm packages 

### PR DESCRIPTION
**Pull request by AhmedElywa**
When we create any PR the action workflow gets an error installing the npm package.
as a fix, I removed the `yarn.lock` file and then installed fresh packages.

#### TODO

- [ ] docs
- [x] tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
   },
   "prettier": "@prisma-labs/prettier-config",
   "peerDependencies": {
-    "@prisma/client": "^3.5",
+    "@prisma/client": "^3.10",
```
Comment (andrewicarlson) [inline package.json:125]: Per the other PR, I believe this is a [breaking change](https://github.com/prisma/nexus-prisma/pull/204/files#r855409916) that we need a plan for communicating if we're going to make it.

Comment (andrewicarlson) [reply package.json:125]: I also see that in the other PR it was bumped to `3.12`, and here it's at `3.10`. Was that intentional?

File: tests/e2e/fixtures/ncc/package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@prisma/client": "__dynamic__",
-    "@vercel/ncc": "0.29.2",
+    "@vercel/ncc": "0.33.4",
```
Comment (andrewicarlson) [inline tests/e2e/fixtures/ncc/package.json:13]: Per the other PR you mentioned we can [remove this dependency change](https://github.com/prisma/nexus-prisma/pull/204/files#r855524488).

File: package.json (hunk 1 above)
Comment (AhmedElywa) [reply package.json:125]: There is no different on the dmmf since 3.10 I added a test for current versions 3.12 and 3.10

Comment (AhmedElywa) [reply package.json:125]: > Per the other PR, I believe this is a [breaking change](https://github.com/prisma/nexus-prisma/pull/204/files#r855409916) that we need a plan for communicating if we're going to make it. 

I don't know why it's breaking change

File: tests/e2e/fixtures/ncc/package.json (hunk 2 above)
Comment (AhmedElywa) [reply tests/e2e/fixtures/ncc/package.json:13]: When I removed it the test not working right

**Comment by andrewicarlson** [comment]
@AhmedElywa if the blocking issue was the `yarn.lock` being outdated (having old URL's that are no longer resolvable) could we just update the `yarn.lock` and leave everything else as is? 

See https://github.com/prisma/nexus-prisma/pull/206

**Pull request closed without merging**